src/inmanta/agent/forking_executor.py

```diff
@@ -175,13 +194,19 @@ def __init__(
         sources: list[inmanta.loader.ModuleSource],
         environment: uuid.UUID,
         uri: str,
+        venv_checkup_interval: typing.Optional[int] = None,
```

Comment:
I don't know if you see another way to change the interval only in the test

Reply:
I think this is fine